pages/linux/getconf.md

```diff
@@ -0,0 +1,24 @@
+# getconf
+
+> getconf - get configuration values from your Linux system.
+> 
+
+- List every configuration values available
```

Comment:
```suggestion
- List [a]ll configuration values available:
```

- This change is optional - no corresponding style guide requirement exists.
- We often provide mnemonics for short options to permit users easy memorize them.